erpnext/accounts/doctype/subscriber/subscriber.json

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
Do we need a new Subscriber doctype? Maybe we can directly use Customer.

Reply:
Actually we need the Subscriber doctype to capture things that are subscription related as time goes on. For example, i'll soon add the ability to use coupons in subscriptions. Coupons can added on subscription basis or on subscriber basis. If the user chooses to add a coupon on subscriber basis, Customer doctype doesn't seem the right place to add it. Another potential addition is account balance. User's should be able to view the Subscriber's balance on account without going through the subscriptions. Customer doctype will also not be a good place for it.

Reply:
Hi @tundebabzy 

Trust you're doing great. I see your point here but I hope the Subscriber concept doesn't get a bit confusing for users. My first inclination was also 'Why doesn't the Subscriber field link straight to the Customer doctype?' After your explanation though, it makes sense

Please include 'Member (Non-Profit Domain)' in the Subscriber DocType too!

Cheers!